include/config/config_debug.h

```diff
@@ -34,4 +37,45 @@
 #define BOOT_CUTSCENE 0x0000
 #define BOOT_PLAYER_NAME 0x15, 0x12, 0x17, 0x14, 0x3E, 0x3E, 0x3E, 0x3E // "LINK"
 
+/**
+ * Disable every debug features
+ */
+// #define DISABLE_DEBUG_FEATURES
+
+// -------------------------------------------
+
+// TODO: find a better way to do that
+#ifndef DEBUG_ROM
+#define DISABLE_DEBUG_FEATURES
+#endif
+
+#ifdef DISABLE_DEBUG_FEATURES
+#undef SHOW_CS_INFOS
+#undef SHOW_INPUT_DISPLAY
+#undef SHOW_TIME_INFOS
+#undef NO_TEST_SCENES
```

Comment:
Wouldn't this be a "retail" feature? Since the test scenes are a debug feature?

Reply:
you mean move ``NO_TEST_SCENES`` in config_game.h? I'm not sure to understand what you mean exactly

Reply:
Why are we *un*defining NO_TEST_SCENES when debug features are off? That would mean that we are including test scenes for retail builds, and not debug builds, it seems.

Reply:
The solution here btw should be to make an `INCLUDE_TEST_SCENES` instead of this. (so then it's not a negative define)

Reply:
oh yea, it's clearly a mistake then, debug scene shouldn't be part of retail